AppKit/CPTextView/CPLayoutManager.j

```diff
@@ -708,18 +708,6 @@ _oncontextmenuhandler = function () { return false; };
     return lineFragment._glyphsOffsets[index];
 }
 
-- (double)_descentAtLocation:(unsigned)location
```

Comment:
please also remove   [_layoutManager _descentAtLocation:loc] in line 2193 of CPTextView.j

Reply:
and lines 2201-2203

Reply:
sorry, i just realized that you already did this